packages/loot-core/src/server/accounts/rules.ts

```diff
@@ -639,14 +639,17 @@ export class RuleIndexer {
 
 const OP_SCORES = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind changing `RuleConditionEntity['op']` from `unknown` to `'is' | 'isNot' | 'oneOf' | ...` so we catch issues such as this one more easily?

```suggestion
type Operations = 'is' | 'isNot' | 'oneOf' | ...;

const OP_SCORES: Record<RuleConditionEntity['op'], number> = {
```